core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -263,6 +263,21 @@
       </div>
     </script>
 
+    <script type="text/ng-template" id="modals/welcomeExploration">
```

Comment:
Some design questions:
- Do you think the modal looks good as a full-screen thing, or would it be better for there to be some of the editor showing around its edges? I'm fine either way, I just wanted to bring up the possibility.
- Would it be better for the modal to just appear all-at-once, rather than slide in from the top? Again, I'm fine either way; just wanted to suggest the possibility.